src/file.js

```diff
@@ -120,4 +123,12 @@ file.linkDirs = function linkDirs(sourceDir, destDir, callback) {
 	fs.symlink(sourceDir, destDir, type, callback);
 };
 
+file.typeToExtension = function (type) {
```

Comment:
If you are adding this function to file then might as well use it from controllers/uploads as well.
https://github.com/NodeBB/NodeBB/blob/master/src/controllers/uploads.js#L231-L237